common/tools/dev-tool/src/config/rollup.base.config.ts

```diff
@@ -149,6 +149,7 @@ export function sourcemaps() {
         debug("no map for file ", id);
         return { code, map: null };
       } catch (e) {
+        // eslint-disable-next-line no-inner-declarations
         function toString(error: any): string {
```

Comment:
It is fine if we move it outside this scope.

Reply:
It's fine here with one usage